src/main/resources/io/snyk/jenkins/SnykReportBuildAction/report.jelly

```diff
@@ -7,17 +7,7 @@
     </l:side-panel>
 
     <l:main-panel>
-      <a href="../${it.urlName}">Back To Reports</a>
-      <br/>
-      <br/>
-      <iframe width="95%" height="700" id="reportsframe" frameborder="0" src=""/>
-      <br/>
-      <br/>
-      <a href="../${it.urlName}">Back To Reports</a>
+      <iframe width="100%" height="100%" id="reportsframe" frameborder="0" src="${it.getArtifactHref()}" style="min-height: 700px;"/>
     </l:main-panel>
-
-    <script>
-      document.getElementById("reportsframe").src = "../artifact/" + location.hash.substr(1)
```

Comment:
This script was in the wrong place (Jelly magic caused it to render ~4 times in different places) and caused null pointer errors on the browser. Instead of moving it to the right place, I changed it to generate the URL server-side where there will be less browser-related problems.